sway/commands/output/transform.c

```diff
@@ -32,6 +33,11 @@ struct cmd_results *output_cmd_transform(int argc, char **argv) {
 	} else {
 		return cmd_results_new(CMD_INVALID, "Invalid output transform.");
 	}
+
+	// Sway uses clockwise transforms, while WL_OUTPUT_TRANSFORM_* describe
+	// anti-clockwise transforms
+	transform = wlr_output_transform_invert(transform);
```

Comment:
While `wlr_output_transform_invert` correctly fixes the rotation direction for pure rotations, the inverse transformation of `WL_OUTPUT_TRANSFORM_FLIPPED_90` is itself, not the value `WL_OUTPUT_TRANSFORM_FLIPPED_270` that the wlroots patch swapped it with.

Reply:
Oh, right. I guess the clockwise/anticlockwise handling needs to be fixed as well.